### ISSUE TITLE
请问是否可以在不下载数据进行训练的情况下，进行demo测试？

### ISSUE THREAD

**Issue by btdan** [Tencent/tencent-ml-images#7]
如题，请问是否可以不直接下载数据进行训练，而是通过您网页中提供的Checkpoints
ckpt-resnet101-mlimages
ckpt-resnet101-mlimages-imagenet    
来进行demo测试？是否辛苦提供一下具体步骤？非常感谢。
Thanks very much

**Comment by wubaoyuan** [comment]
@btdan  Maybe you can check './example/finetune.sh'? Just replace the ImageNet images with your own images, and generate the tfrecord file using './tfrecord.sh'

**Comment by btdan** [comment]
非常感谢您的回复。我的意思是，是否有现成的模型来直接对某张图像进行测试。我主要是想了解一下对于小目标物体的识别效果？恳请您能否告诉我具体的操作步骤？非常感谢。
Thanks very much.

**Comment by wubaoyuan** [comment]
@btdan I get it. We will add such a demo soon. But you can firstly check the similar demo is `finetune.sh`, which loads the checkpoint, and extract the features of the image.

**Comment by wubaoyuan** [comment]
@btdan It is a good suggestion :)

**Comment by btdan** [comment]
非常感谢，很期待您的更新。多谢

**Comment by wubaoyuan** [comment]
@btdan the demo for image classification has been added. Please check the updated README

**Comment by Jason-xin** [comment]
@wubaoyuan The demo is based on imagenet dictionary, can you provide a demo based on ML-Image dictionary??  You have provided two pretrained model, are both trained on ImageNet dictionary?? instead of ML-Image dictionary?
![image](https://user-images.githubusercontent.com/13817995/48941771-db957b00-ef57-11e8-9fb4-0bd73d1a488a.png)

**Comment by wubaoyuan** [comment]
@Jason-xin Hi, the first ckpt is pre-trained with the ML-Images dictionary. You suggestion is good, we plan to add a demo to use this ckpt.

**Comment by Jason-xin** [comment]
@wubaoyuan Thanks，maybe it just need add ML-Images dictionary text file? If not, hope your new demo in ML-Images soon.

**Comment by wubaoyuan** [comment]
@Jason-xin
  Yes. As the output layers of ResNet for both ML-Images and ImageNet are M (size of dictionary) sigmoid functions, one just need to replace the dictionary.

**Comment by Jason-xin** [comment]
@wubaoyuan Thank you! I have done~ Hope your paper ^-^

**Comment by Easyfeng222** [comment]
@wubaoyuan  Excuse me, When can I open the test code for ML-images? Looking forward to your reply

**Comment by Easyfeng222** [comment]
@Jason-xin Excuse me, have you completed the ML-images test?

**Comment by Easyfeng222** [comment]
@Jason-xin Excuse me, have you completed the ML-images test? If you can, can you talk about the specific method? Thank you in advance.

**Comment by Jason-xin** [comment]
@Easyfeng222 I just test the single-label classification on ML-Images, you can just change "--dictionary" param in  /example/image_classification.sh.
